unison-src/transcripts/mergeloop.output.md

```diff
@@ -136,9 +136,8 @@ b = 2
   namespace hash.
   
   #0lf1cvdccp
-  #2quan5n72t
-  #gjfovomom2
+  #ofcsecdak0
   ⑂
-  ⊙ #bpfcuip0ru
+  ⊙ #0ucrusr0bl
```

Comment:
Why did it change from `Merge [#olf1, #2qua, #gjfo]` to `Merge [#olf1, #bpfc]?

Reply:
Realizing that merges are not associative, all merges are now three-way rather than n-way